pages/partials/navbarPlain.ejs

```diff
@@ -1,4 +1,5 @@
 
       <% if (is_administrator) { %>
       <li class="nav-item <% if (navPage == "admin") { %>active<% } %>"><a class="nav-link" href="<%= plainUrlPrefix %>/administrator/overview">Admin</a></li>
+      <li class="nav-item <% if (navPage == "adminQueries") { %>active<% } %>"><a class="nav-link" href="<%= plainUrlPrefix %>/administrator/queries">Queries</a></li>
```

Comment:
To me it seems a bit strange to have an "admin" tab and a "queries" tab when they're both admin tasks -- if we don't expect to have too many of these queries could we combine them into one page?

Reply:
Totally agreed! I'd actually like to break up the Admin page into sub-pages for all the individual panels, and add a tab-bar under "Admin" for the sub-pages. Then "Queries" would be one of these tabs.

I didn't want to make this PR more complex by also doing this breakup, but I'll add an issue for it now.